src/backends/perf_pt/mod.rs

```diff
@@ -277,12 +275,20 @@ impl Tracer for PerfPTTracer {
             return Err(HWTracerError::TracerAlreadyStarted);
         }
 
+        // At the time of writing, we have to use a fresh Perf file descriptor to ensure traces
```

Comment:
I think the second sentence ("this is required") is redundant?

Reply:
I think it's OK as-is. A trace not starting with a `PSB+` is a valid trace, but it may not decode.

Reply:
How can it be valid if it can't be decoded?

Reply:
There are many levels. The packets decoder would succeed, but the instruction and block decoders would not.

Reply:
So it's a valid trace at the packet level, if you see what I mean?

Reply:
Ah, so maybe we need to say which decoder wouldn't work in the comment?